www/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: [
+    '@pluralsh/eslint-config-typescript',
```

Comment:
That's actually a bit weird that it works for JS|JSX files since the config only supports files with extensions `ts` and `tsx`.